### PR TITLE
Music Changes

### PR DESCRIPTION
**Pull request by THEb0nny**
Added group names to ns.ts, corresponding groups are specified for blocks. And some clarifying changes have been made. Something unnecessary has been removed.

Before change
image

After the changes
image

### CODE REVIEW

**Comment by THEb0nny** [comment]
@jwunderl look here please

File: libs/music/music.cpp
Comment (jwunderl) [inline libs/music/music.cpp:1]: Please undo the indentation change in this file, our typical style for cpp doesn't indent on namespaces / no need to change here / makes tracking changes hard (e.g. this looks like a significant change even though it's only 4 extra lines)

**Review by jwunderl — CHANGES_REQUESTED** [review]
Please undo indentation change in music.cpp, otherwise LGTM~

**Review by jwunderl — APPROVED** [review]
LGTM thanks!

**Pull request merged**